assets/backend/store/appointments/appointments.ts

```diff
@@ -68,6 +89,10 @@ export const reducer = (state = DEFAULT_APPOINTMENTS_STATE, action: Action) => {
 		case 'SET_APPOINTMENTS':
 			return produce(state, (draft) => {
 				draft.appointments = action.appointments;
+				draft.totalItems = action.totalItems;
+				draft.totalPages = action.totalPages;
+				draft.postsPerPage = action.postsPerPage;
+				draft.currentPage = action.currentPage;
 			});
```

Comment:
Since we are just rewriting one object to another 1:1 with the same props, this could be simplified to just:

```ts
return produce(state, (draft) => {
    return action;
});
```

Additionally to modifying the `draft` state object, we can also return completely new state from the `produce` callback. https://immerjs.github.io/immer/return/